javaHapiValidatorLambda/src/main/java/software/amazon/fwoa/Utils.java

```diff
@@ -0,0 +1,28 @@
+package software.amazon.fwoa;
+
+import java.util.Collections;
+import java.util.List;
+
+public final class Utils {
+    public static class IGObject {
+        private String key;
+        private String content;
+
+        public IGObject(String key, String content) {
+            this.key = key;
+            this.content = content;
+        }
+
+        public String getKey() {
+            return key;
+        }
+
+        public String getContent() {
+            return content;
+        }
+    }
+
+    public static List<IGObject> EmptyIfNull(List<IGObject> iterable) {
```

Comment:
Where is this function used?

Reply:
This is used in the Validator.java file, lines 159 and 165